sql/load_sql_context.sql

```diff
@@ -15,6 +15,17 @@ schemas_(oid, name) as (
             pn.oid,
             'USAGE'
         )
+),
+type_hierarchy(base_oid, current_oid) AS (
+    SELECT oid, oid
+    FROM pg_type
+    WHERE typbasetype = 0
+
+    UNION ALL
+
+    SELECT th.base_oid, pt.oid
+    FROM pg_type pt
+    JOIN type_hierarchy th ON pt.typbasetype = th.current_oid
```

Comment:
nit: match existing SQL formatting style (lowercase w/ newlines)

Reply:
Sure, thing. 

Just thinking out loud  but it might be a good idea to add a formatter for the sql code. I haven't used pre-commit before so I don't know how difficult it'll be to add. But what about adding [pgFormatter](https://www.postgresql.org/about/news/pgformatter-54-released-2569/) until the supabase/postgres_lsp can handle formatting?

Reply:
yeah a formatter would be great but I haven't found one that I'm happy with. they tend to mangle unusual stuff like hierarchies of `jsonb_build_object`

throw the contents of that file into https://sqlformat.darold.net to see for yourself

Reply:
Point taken, that was unreadable.